tests/bluetooth/host/keys/bt_keys_get_addr/src/test_suite_full_list_no_overwrite.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include "testing_common_defs.h"
+#include <addr.h>
+#include <keys.h>
+#include "mocks/hci_core_expects.h"
+
+/*
+ *  Test adding extra (ID, Address) pair while the keys pool list is full
+ *
+ *  Constraints:
+ *   - Keys pool list is full
```

Comment:
I don't see where this is done, I would expect a setup function run before or at the start of the test suite that actually does this.
If you're relying on state from a previous state, that's not ok.